From 323c86533fd3bc0d41b9df5f3ccce562b1da2da4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=99=88=E6=96=87?= <chenwen54@huawei.com>
Date: Sat, 14 Mar 2026 11:48:02 +0800
Subject: [PATCH] urma: for updates to external files or APIs, minor version
 numbers need to be upgraded

(cherry picked from commit 799b824cb0c08591d786e9eedba42d74f6cb6c59)
---
 src/urma/lib/urma/bond/CMakeLists.txt | 5 ++++-
 src/urma/lib/urma/core/CMakeLists.txt | 2 ++
 src/urma/lib/uvs/core/CMakeLists.txt  | 3 +++
 3 files changed, 9 insertions(+), 1 deletion(-)

diff --git a/src/urma/lib/urma/bond/CMakeLists.txt b/src/urma/lib/urma/bond/CMakeLists.txt
index d666efbb..8acd01b5 100644
--- a/src/urma/lib/urma/bond/CMakeLists.txt
+++ b/src/urma/lib/urma/bond/CMakeLists.txt
@@ -9,6 +9,9 @@ add_library(urma_ubagg SHARED
     ${PROVIDER_AGG_UTILS_DIR_SRCS}
 )
 
+set(PROJECT_VERSION "0.0.2")
+set(PROJECT_VERSION_MAJOR "0")
+
 set_target_properties(urma_ubagg
     PROPERTIES
         C_STANDARD 99
@@ -32,4 +35,4 @@ install(TARGETS urma_ubagg
 install(FILES
     ${CMAKE_CURRENT_SOURCE_DIR}/include/urma_ubagg.h
     DESTINATION /usr/include/ub/umdk/urma
-)
\ No newline at end of file
+)
diff --git a/src/urma/lib/urma/core/CMakeLists.txt b/src/urma/lib/urma/core/CMakeLists.txt
index a5eb7824..8e988d71 100644
--- a/src/urma/lib/urma/core/CMakeLists.txt
+++ b/src/urma/lib/urma/core/CMakeLists.txt
@@ -10,6 +10,8 @@ add_library(urma SHARED
 add_library(urma_static STATIC
     ${URMA_DIR_SRCS}
 )
+set(PROJECT_VERSION "0.0.2")
+set(PROJECT_VERSION_MAJOR "0")
 
 set_target_properties(urma
     PROPERTIES
diff --git a/src/urma/lib/uvs/core/CMakeLists.txt b/src/urma/lib/uvs/core/CMakeLists.txt
index dc1e512b..13f7717e 100644
--- a/src/urma/lib/uvs/core/CMakeLists.txt
+++ b/src/urma/lib/uvs/core/CMakeLists.txt
@@ -7,6 +7,9 @@ add_library(tpsa SHARED
     ${TPSA_SO_DIR_SRCS}
 )
 
+set(PROJECT_VERSION "0.0.2")
+set(PROJECT_VERSION_MAJOR "0")
+
 set_target_properties(tpsa
     PROPERTIES
         C_STANDARD 99
-- 
GitCode

